app/routes/surveys/components/Submissions/Results.js

```diff
@@ -87,8 +135,13 @@ const Results = ({
           const pieColors = CHART_COLORS.filter(
             (color, i) => !colorsToRemove.includes(i)
           );
+          const ak = graphOptions.find(a => a.value === question.displayType);
+          const barData = graphData[question.id];
           const labelRadius = pieData.length === 1 ? -10 : 60;
-
+          const highestSubmissionsCount =
+            barData.length > 0
+              ? barData.reduce((a, b) => Math.max(a, b.selections), 0)
+              : 0;
```

Comment:
Do you need this `barData.length > 0`? If `barData === []`, the `reduce` will just result in the start value, which is 0.
```
const a = [];
a.reduce((a, b) => a + b, 0);  // 0
```